File: C#/TopflytechCodec/TopflytechCodec/Entities/Rs232TireMessage.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace TopflytechCodec.Entities
{
    class Rs232TireMessage :Rs232DeviceMessage
    {
        private String sensorId;

        public String SensorId
        {
            get { return sensorId; }
            set { sensorId = value; }
        }
        private double voltage = 0;

        public double Voltage
        {
            get { return voltage; }
            set { voltage = value; }
        }
        private double airPressure = 0;

        public double AirPressure
        {
            get { return airPressure; }
            set { airPressure = value; }
        }
        private int airTemp = 0;

        public int AirTemp
        {
            get { return airTemp; }
            set { airTemp = value; }
        }
        private int status = 0;

        public int Status
        {
            get { return status; }
            set { status = value; }
        }
    }
}

File: Java/topflytechCodec/src/com/topflytech/codec/entities/Rs232TireMessage.java
package com.topflytech.codec.entities;

/**
 * Created by admin on 2017/11/27.
 */
public class Rs232TireMessage extends Rs232DeviceMessage {

    public String getSensorId() {
        return sensorId;
    }

    public void setSensorId(String sensorId) {
        this.sensorId = sensorId;
    }

    public double getVoltage() {
        return voltage;
    }

    public void setVoltage(double voltage) {
        this.voltage = voltage;
    }

    public double getAirPressure() {
        return airPressure;
    }

    public void setAirPressure(double airPressure) {
        this.airPressure = airPressure;
    }

    public int getAirTemp() {
        return airTemp;
    }

    public void setAirTemp(int airTemp) {
        this.airTemp = airTemp;
    }

    public int getStatus() {
        return status;
    }

    public void setStatus(int status) {
        this.status = status;
    }

    private String sensorId;
    private double voltage = 0;
    private double airPressure = 0;
    private int airTemp = 0;
    private int status = 0;
}
